src/schneider_lab_to_nwb/schneider_2024/schneider_2024_convert_session.py

```diff
@@ -12,14 +12,19 @@
 from schneider_lab_to_nwb.schneider_2024 import Schneider2024NWBConverter
 
 
-def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], stub_test: bool = False):
-
-    data_dir_path = Path(data_dir_path)
+def session_to_nwb(
+    recording_folder_path: Union[str, Path],
```

Comment:
You can use str | Path instead of Union[str, Path] if you prefer btw.

Reply:
right! I'm still used to catering to python 3.9, but this conversion need not be so restricted.